src/lib/Header.svelte

```diff
@@ -65,7 +70,7 @@
         <div id="logo"></div>
         <div id="menubar">
             {#each items as item}
-                <Dropdown menu={item}></Dropdown>
+                <Dropdown menu={item} getState={() => menuOpen} onClick={toggleMenu}></Dropdown>
```

Comment:
since the Dropdown element has no access to modify the parent's (Header) variables, I had to wrap the toggle function and pass it to the Dropdown when it's made, that's the onClick method.